metastore/src/java/org/apache/hadoop/hive/metastore/HiveMetaStoreUtils.java

```diff
@@ -201,7 +201,7 @@ static public Deserializer getDeserializer(Configuration conf,
 
   private static final String FROM_SERIALIZER = "from deserializer";
   private static String determineFieldComment(String comment) {
-    return (comment == null) ? FROM_SERIALIZER : comment;
+    return ("".equals(comment)) ? null : comment;
```

Comment:
I think we should not specify a value if the comment is null, so I modified the unit test.
Is my understanding right?